indy_client/cli/cli.py

```diff
@@ -1912,8 +1917,13 @@ def getActiveEnv(self):
                                                 self.currPromptText)
         return env
 
+    def get_available_networks(self):
+        return [dir for dir in os.listdir(self.ledger_base_dir)
```

Comment:
`dir` is a built-in function, so it's better to choose another name for the variable

Reply:
done